cardano-crypto-class/src/Cardano/Crypto/DSIGN/Ed448.hs

```diff
@@ -108,3 +119,9 @@ decodeBA f = do
   case f bs of
     CryptoPassed a -> return a
     CryptoFailed e -> fail $ "decodeBA: " ++ show e
+
+
+cryptoFailableToMaybe :: CryptoFailable a -> Maybe a
+cryptoFailableToMaybe (CryptoPassed a) = Just a
+cryptoFailableToMaybe (CryptoFailed _) = Nothing
```

Comment:
This function is copy-pasted from `DSIGN/Ed25519.hs`. Can it live in `Cardano.Crypto.Util`?